content/unexpected-reboots.md

```diff
@@ -0,0 +1,441 @@
+---
+title: Reboots (Unexpected)
+description: >
+   If your system is spontaneously rebooting, this article provides steps to isolate the cause.
+keywords:
+  - reboot
+  - shutdown
+  - restart
+  - reset
+  - crash
+  - reinstall
+  - refresh
+  - power-off
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: hardware-troubleshooting
+tableOfContents: true
+---
+
+If your system is spontaneously rebooting, this article provides steps to isolate the cause.
+
+> **NOTE:** There is a subtle but important difference between a reboot and a shutdown/power-off. In a reboot, parts of the system are left powered on to facilitate restarting the system. In a shutdown/power-off, hardware from all parts of the system except charging circuitry and capacitors is powered down.
+
+## Troubleshooting (Device Agnostic)
+
+### Software Causes
+
+Anything not directly affected by hardware, is controlled by software. This includes the kernel, drivers, and the desktop environment. The Operating System (OS), is essentially a software program that controls other software programs.
+
+If anything running in software causes a large enough cascade effect, it can take the rest of the system down and cause reboots or shut-offs.
+
+Software factors that can cause spontaneous reboots or crashes, include but are not limited to:
+
+1. Kernel Panics
+2. Kernel or Desktop Process Crashes
+3. Application Memory Leaks
+4. Driver Errors
+5. User Configuration Error
+6. Corrupted Operating System Files
+
+### Possible Solutions
+
+<!-- To be added ASAP
+
+**Upgrade or Reinstall the Kernel**
+
+[LINK]
+
+-->
+
+#### Check for Memory Leaks
+
+If an app is taking more than its fair share of system memory, that can have a runaway affect resulting in shutdowns or restarts.
```

Comment:
"affect" should be "effect." I would probably also avoid using "that" in the same sentence as what it's referring to, and I would add a comma before "resulting in" since those words start a parenthetical phrase.

"If an app is taking more than its fair share of system memory, it can have a runaway effect, resulting in shutdowns or restarts."

Reply:
I do know this! Haha. I think in my head I wrote "apps affect..."

Reply:
Fixed.